### PR TITLE
AC-1165 no data found displayed whilst loading users roles

### PR DESCRIPTION
**Pull request by lillie-dae**
### Description

Fixes AC-1165 .

issues with tables not showing skeleton loaders

### Status
**Ready**

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New unit tests added to cover the changes.
- [ ] New e2e tests added to cover the changes. (If it has been decided these will not be added with the PR, a seperate ticket **must** be created and linked in the ticket below before merge)
- [ ] All tests passed locally.

### CODE REVIEW

**Comment by samrooke** [comment]
Why are we adding 6 second timeouts to all the the mock responses?

File: src/components/user-management/UserRolesTabItem.vue [hunk 1]
```diff
@@ -16,7 +16,7 @@
 
 <template>
     <v-tab-item>
-        <div v-if="!loading && rolesList !== undefined">
+        <div>
```
Comment (woodheadio) [inline src/components/user-management/UserRolesTabItem.vue:19]: If there are no longer any conditions on this div do we need it at all?

File: src/components/user-management/UserTabItem.vue [hunk 2]
```diff
@@ -16,7 +16,7 @@
 
 <template>
     <v-tab-item>
-        <div v-if="!loading && users !== undefined">
+        <div>
```
Comment (woodheadio) [inline src/components/user-management/UserTabItem.vue:19]: Can we remove this now redundant div?

File: src/views/Workflows.vue [hunk 3]
```diff
@@ -29,14 +28,15 @@
                     <v-icon small class="ml-1">mdi-plus</v-icon>
                 </v-btn>
             </div>
-            <div v-if="!loading && workflowPage !== undefined">
+            <div>
```
Comment (woodheadio) [inline src/views/Workflows.vue:31]: Can we remove this now redundant div?

**Review by woodheadio — APPROVED** [review]

**Comment by lillie-dae** [comment]
> Why are we adding 6 second timeouts to all the the mock responses?

so if people make changes in future they'll see if it affects the table showing 'no workflows data' rather than the skeleton loader.

File: src/components/user-management/UserTabItem.vue (hunk 2 above)
Comment (lillie-dae) [reply src/components/user-management/UserTabItem.vue]: see comment: [UserRolesTabItem.vue](https://github.com/AI4VBH/AIDE-front-end/pull/85/files/38c5bbb2694778639a6532a0ddd2c9792eafd7a7#diff-49fccd610f796f6f6ea8927c46a43dc1d8484c6e465d4650042fa30957baab3e)

File: src/views/Workflows.vue (hunk 3 above)
Comment (lillie-dae) [reply src/views/Workflows.vue]: see comment: [UserRolesTabItem.vue](https://github.com/AI4VBH/AIDE-front-end/pull/85/files/38c5bbb2694778639a6532a0ddd2c9792eafd7a7#diff-49fccd610f796f6f6ea8927c46a43dc1d8484c6e465d4650042fa30957baab3e)

**Review by alexjbazin — APPROVED** [review]

**Pull request merged**